packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -2219,6 +2221,12 @@ class TimePickerDialog extends StatefulWidget {
   /// Callback called when the selected entry mode is changed.
   final EntryModeChangeCallback? onEntryModeChanged;
 
+  /// {@macro flutter.material.time_picker.switchToInputEntryModeIcon}
+  final Icon switchToInputEntryModeIcon;
+
+  /// {@macro flutter.material.time_picker.switchToTimerEntryModeIcon}
+  final Icon switchToTimerEntryModeIcon;
```

Comment:
I was going to leave a suggestion here, but since this is set up to mirror the date picker, it's probably best to leave as-is for consistency.